packages/peregrine/lib/talons/CartPage/ProductListing/EditModal/useProductForm.js

```diff
@@ -163,3 +182,19 @@ export const useProductForm = props => {
         isSaving
     };
 };
+
+/** JSDocs type definitions */
+
+/**
+ * Object type returned by the {@link useProductForm} talon.
+ * It provides props data for a product form UI component inside a modal.
+ *
+ * @typedef {Object} ProductFormProps
+ *
+ * @property {Object} configItem Cart item to configure
+ * @property {Array<Object>} formErrors An array of form errors resulting from a configuration or quantity value
```

Comment:
```suggestion
 * @property {Array<Error>} formErrors An array of form errors resulting from a configuration or quantity value
```

All of the `formErrors` returns should be the same type. Can you double check? I think you also updated `useShippingForm` but omitted the array type.